wolfcrypt/user-crypto/src/rsa.c

```diff
@@ -1930,7 +1930,7 @@ int wc_RsaFlattenPublicKey(RsaKey* key, byte* e, word32* eSz, byte* n,
     if (key == NULL || e == NULL || eSz == NULL || n == NULL || nSz == NULL)
        return USER_CRYPTO_ERROR;
 
-    bytSz = sizeof(byte);
+    bytSz = sizeof(byte) * 8;
```

Comment:
The variable bytSz is used to convert a size in bits to a size in bytes (and it does it badly). The comments say it and the usage indicates that's what it is attempting to do.